src/main/java/com/microsoft/azure/spring/data/cosmosdb/repository/support/SimpleDocumentDbRepository.java

```diff
@@ -53,41 +54,44 @@ private void createCollectionIfNotExists() {
     /**
      * save entity without partition
      *
-     * @param entity to be saved
+     * @param domain to be saved
      * @param <S>
      * @return entity
      */
     @Override
-    public <S extends T> S save(S entity) {
-        Assert.notNull(entity, "entity must not be null");
+    public <S extends T> S save(S domain) {
```

Comment:
Why change from `entity` to `domain`, I prefer `entity` as it represents data, but `domain` is an abstraction?

Reply:
I prefer the naming of `domain` instead of `entity`, we can have a discuss about this and align the naming in another PR.